googleapis/google-cloud-node
# migrate code from googleapis/nodejs-access-approval

- chore: add initial files for launch
- chore: add initial files for launch
- fix: header
- chore: add initial files for launch
- feat!: initial stub of library
- feat!: add initial samples (#5)
- chore: release 1.0.0 (#1)
- build: package name, and nits (#9)
- chore: update flakybot config and package name in docs (#15)
- chore: update flakybot config and a few docs (#16)
- build: config for owlbot (#17)
- build: update gapic-generator-typescript to v1.2.10. (#19)
- chore: migrate to owl-bot with staging dir (#20)
- chore(deps): update dependency sinon to v10 (#24)
- chore(deps): update dependency @types/sinon to v10 (#31)
- fix(deps): require google-gax v2.12.0 (#40)
- chore: new owl bot post processor docker image (#42)
- fix: use require() to load JSON protos (#43)
- chore: update gapic-generator-typescript to v1.3.2 (#44)
- chore: release 1.0.1 (#41)
- chore(deps): update dependency sinon to v11 (#47)
- fix: GoogleAdsError missing using generator version after 1.3.0 (#48)
- chore: release 1.0.2 (#49)
- fix: make request optional in all cases (#55)
- chore: release 1.0.3 (#56)
- fix(deps): google-gax v2.17.0 with mTLS (#59)
- chore: release 1.0.4 (#60)
- fix(deps): google-gax v2.17.1 (#62)
- chore: release 1.0.5 (#63)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#64)
- chore: release 1.0.6 (#65)
- build: update auto-approve config for new validation (#1169) (#68)
- chore(nodejs): update client ref docs link in metadata (#70)
- fix(build): migrate to using main branch (#72)
- chore: release 1.0.7 (#73)
- fix(deps): google-gax v2.24.1 (#74)
- chore: release 1.0.8 (#75)
- feat: turns on self-signed JWT feature flag (#76)
- chore: release 1.1.0 (#77)
- chore: relocate owl bot post processor (#80)
- build(node): update deps used during postprocessing (#1243) (#85)
- chore(cloud-rad): delete api-extractor config (#86)
- chore(deps): update dependency @types/node to v16 (#87)
- chore(deps): update dependency sinon to v12 (#88)
- docs(samples): add example tags to generated samples (#89)
- build: add generated samples to .eslintignore (#90)
- docs(node): support "stable"/"preview" release level (#1312) (#93)
- chore: add api_shortname and library_type to repo metadata (#92)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#95)
- test(nodejs): remove 15 add 16 (#1322) (#97)
- build: update copyright year to 2022 (#99)
- chore: update v2.12.0 gapic-generator-typescript (#101)
- chore(deps): update dependency sinon to v13 (#103)
- docs(nodejs): version support policy edits (#1346) (#105)
- docs: added resource annotations (#106)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#107)
- chore: update v2.14.2 gapic-generator-typescript (#111)
- build(node): update client library version in samples metadata (#1356) (#118)
- chore(deps): update dependency @types/mocha to v9 (#119)
- chore(deps): update dependency sinon to v14 (#122)
- build!: update library to use Node 12 (#124)
- feat: update protos to include InvalidateApprovalRequest and GetAccessApprovalServiceAccount APIs (#126)
- chore(main): release 2.0.0 (#125)
- chore(deps): update dependency jsdoc-fresh to v2 (#128)
- chore(deps): update dependency jsdoc-region-tag to v2 (#129)
- feat: support regapic LRO (#130)
- chore(main): release 2.1.0 (#131)
- chore(deps): update dependency linkinator to v4 (#134)
- fix: better support for fallback mode (#135)
- fix: change import long to require (#136)
- fix: do not import the whole google-gax from proto JS (#1553) (#140)
- fix: allow passing gax instance to client constructor (#139)
- chore(main): release 2.1.1 (#137)
- fix: preserve default values in x-goog-request-params header (#141)
- test: use fully qualified request type name in tests (#144)
- fix(deps): use google-gax v3.5.2 (#148)
- chore(deps): update dependency jsdoc to v4 (#150)
- chore(deps): update dependency @types/node to v18 (#146)
- fix: regenerated protos JS and TS definitions (#151)
- chore(main): release 2.1.2 (#142)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
